Demo.ipynb

```diff
@@ -125,29 +120,32 @@
     "## Sensitivity analysis: Varying Irradiances\n",
     "\n",
     "pv_mod_g1 = pv_model_class.PvModel(Iscn, Vocn, Imp, Vmp, Kv, Ki, Ns, Gn, 800, Tn, Tn, Egap, err)\n",
-    "pv_mod_g2 = pv_model_class.PvModel(Iscn, Vocn, Imp, Vmp, Kv, Ki, Ns, Gn, 400, Tn, Tn, Egap, err)\n",
-    "v_g1, i_g1, P_g1 = pv_mod_g1.power(Rs, Rp, a)\n",
-    "v_g2, i_g2, P_g2 = pv_mod_g2.power(Rs, Rp, a)\n",
+    "pv_mod_g2 = pv_model_class.PvModel(Iscn, Vocn, Imp, Vmp, Kv, Ki, Ns, Gn, 600, Tn, Tn, Egap, err)\n",
+    "pv_mod_g3 = pv_model_class.PvModel(Iscn, Vocn, Imp, Vmp, Kv, Ki, Ns, Gn, 400, Tn, Tn, Egap, err)\n",
+    "v_g1, i_g1, P_g1 = pv_mod_g1.calculate_power(Rs, Rp, a)\n",
+    "v_g2, i_g2, P_g2 = pv_mod_g2.calculate_power(Rs, Rp, a)\n",
+    "v_g3, i_g3, P_g3 = pv_mod_g3.calculate_power(Rs, Rp, a)\n",
     "\n",
     "plt.plot(v, i, 'b', label='G = 1000')\n",
     "plt.plot(v_g1, i_g1, 'g', label='G = 800')\n",
-    "plt.plot(v_g2, i_g2, 'r', label='G = 400')\n",
+    "plt.plot(v_g2, i_g2, 'r', label='G = 600')\n",
+    "plt.plot(v_g3, i_g3, 'y', label='G = 400')\n",
     "plt.xlabel('Voltage')\n",
     "plt.ylabel('Current')\n",
-    "plt.title('IV Curve with Varying Irradiances')\n",
+    "plt.title('I-V Curve with Varying Irradiances')\n",
     "plt.legend()\n",
+    "plt.axis((None, None, 0, None))\n",
```

Comment:
What does this line do? Does this have to deal with axis limits or something like that?

Reply:
Oh ok that makes sense, is there any reason you specified the third argument as zero instead of another one?

Reply:
Yeah! This function is formatted as follows: `plt.axis((x_axis_min, x_axis_max, y_axis_min, y_axis_max))`. Since I wanted to see the x-axis crossing (which is essentially `y = 0`) in order to see the `Voc` values, I set `y_axis_min = 0`.

Reply:
Yeah, I was glad to stumble upon that. In the past, I've managed `plt.axis` by creating a `plt.plot` object (in many examples, you'll see people using something like `axes`) and updating accordingly. However, I like this more because it gives you more flexibility and works more easily with plotting `pandas.DataFrames`.